.github/workflows/dev-short-tests.yml

```diff
@@ -200,6 +200,18 @@ jobs:
         make clean && make -j all MOREFLAGS="-Werror -DZSTD_NO_INLINE -DZSTD_STRIP_ERROR_STRINGS"
         make clean && make check MOREFLAGS="-Werror -DZSTD_NO_INLINE -DZSTD_STRIP_ERROR_STRINGS"
 
+# This test currently fails on Github Actions specifically.
```

Comment:
I think this is probably because GH Actions runners are not ttys. I believe there is a third-party GH action that creates a tty now, so I can go back and migrate the tests that were failing because it wasn't a tty at some point.